### PR TITLE
Report module prefactor

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

File: corehq/apps/app_manager/suite_xml/features/mobile_ucr.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+                    Field(
+                        header=Header(
+                            text=Text(
+                                locale=Locale(id=id_strings.report_last_sync())
+                            )
+                        ),
+                        template=Template(
+                            text=Text(
+                                xpath=Xpath(
+                                    function="format-date(date(instance('reports')/reports/@last_sync), '%Y-%m-%d %H:%M')"
```
Comment (dimagimon) [inline corehq/apps/app_manager/suite_xml/features/mobile_ucr.py]: E501 line too long (122 > 115 characters)

**Comment by dannyroberts** [comment]
@czue only the first commit removed something (unused), everything else was just tearing out code from the class with no other changes

**Comment by snopoke** [comment]
:+1

**Comment by czue** [comment]
nice

**Pull request merged**